pythonforandroid/build.py

```diff
@@ -694,7 +697,8 @@ def run_pymodules_install(ctx, modules, project_dir, ignore_setup_py=False):
                     _env=copy.copy(env))
 
         # Afterwards, run setup.py if present:
-        if project_has_setup_py(project_dir) and not ignore_setup_py:
+        if project_dir is not None and \
```

Comment:
I would prefer using () instead of \ for line breaks.

Reply:
oops yea that's reasonable, changed it!